subcommands/keys/ca_rotate_tls.go

```diff
@@ -0,0 +1,51 @@
+package keys
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+	"github.com/foundriesio/fioctl/x509"
+)
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "rotate-tls <PKI Directory>",
+		Short: "Rotate the TLS certificate used by Device Gateway and OSTree Server",
+		Run:   doRotateTls,
+		Args:  cobra.ExactArgs(1),
+	}
+	caCmd.AddCommand(cmd)
+	// HSM variables defined in ca_create.go
+	cmd.Flags().StringVarP(&hsmModule, "hsm-module", "", "", "Load a root CA key on a PKCS#11 compatible HSM using this module")
+	cmd.Flags().StringVarP(&hsmPin, "hsm-pin", "", "", "The PKCS#11 PIN to log into the HSM")
+	cmd.Flags().StringVarP(&hsmTokenLabel, "hsm-token-label", "", "", "The label of the HSM token containing the root CA key")
+}
+
+func doRotateTls(cmd *cobra.Command, args []string) {
+	factory := viper.GetString("factory")
+
+	subcommands.DieNotNil(os.Chdir(args[0]))
```

Comment:
Does it matter where exactly this `Chdir` happens to? Can there be invalid targets?